guides/v2.2/config-guide/bootstrap/magento-modes.md

```diff
@@ -47,9 +44,21 @@ You can run Magento in any of the following *modes*:
 				<li><b>Does not allow you to enable or disable cache types in Magento Admin.</b> <a href="{{ page.baseurl }}/config-guide/cli/config-cli-subcommands-cache.html#config-cli-subcommands-cache-en">More information about enabling and disabling the cache</a>.</li>
 			</ul></td>
 	</tr>
+	<tr>
+		<td>maintenance</td>
+		<td><p>Intended to prevent access to the Magento application while it is being updated or reconfigured, this mode:</p>
+			<ul><li>Displays a default <code>Service Temporarily Unavailable</code> page when customers visit store URLs.</li>
+				<li>When the application is in maintenance mode, the <code>var/</code> directory contains the       <code>.maintenance.flag</code> file.</li>
+				<li>You can configure maintenance mode to allow visitor access from a specified list of IP addresses.</li>
+				<li>You can configure a custom maintenance page to display when the Magento application enters maintenance mode.</li>
+			</ul></td>
+		</tr>
 </tbody>
 </table>
 
+{:.bs-callout .bs-callout-info}
+[{{site.data.var.ece}}]({{ page.baseurl }}/cloud/bk-cloud.html) supports only the production and maintenance modes.
+
 ## Default mode
 
 As its name implies, default mode is how the Magento software operates if no other mode is specified. Default mode enables you to deploy the Magento application on a single server without changing any settings. However, default mode is not as optimized for production as is production mode.
```

Comment:
This comment just shows the use of different references to Magento. "Magento software" hmmm. I'm sure there is an inconsistency across docs about this. Application? Software? Product? ugh. @jeff-matthews What say you?

Reply:
I think of Magento as a platform or framework, so "Magento software" is awkward for me. I wouldn't refer to "Node.js software" or "Ruby on Rails software"; I would simply refer to it as Node, Ruby on Rails, or Magento. This is a good topic for team conversation about style.